rust/core-lib/src/file.rs

```diff
@@ -42,6 +44,8 @@ pub struct FileManager {
     /// A monitor of filesystem events, for things like reloading changed files.
     #[cfg(feature = "notify")]
     watcher: FileWatcher,
+    #[cfg(feature = "notify")]
```

Comment:
I don't think this needs to be behind the feature flag; if the feature doesn't exist this will still compile fine, it just won't be used (which is fine).

Edit:

Also, I'm not sure if this makes sense here? If this is on it would mean that it's on for all files?

Reply:
Since we don't want to have a global tail flag, Im going to remove this. Moving it to `FileInfo`.